internal/beater/java_attacher/java_attacher.go

```diff
@@ -354,15 +367,25 @@ func (j *JavaAttacher) verifyJVMExecutable(ctx context.Context, jvm *jvmDetails)
 // once the agent is attached or the context is closed; it may be blocking for long and should be called with a timeout context.
 func (j *JavaAttacher) attachJVM(ctx context.Context, jvm *jvmDetails) error {
 	cmd := j.attachJVMCommand(ctx, jvm)
+	if cmd == nil {
+		return nil
+	}
```

Comment:
why would the cmd be nil?

Reply:
It may if the attacher jar could not be created in the tmp dir.
This is how we avoid trying to copy and issuing errors again and again for all JVMs ran by a user for which we cannot attach.